src/EnergyPlus/DataSurfaces.hh

```diff
@@ -167,6 +167,15 @@ namespace DataSurfaces {
         Num // The counter representing the total number of surface class, always stays at the bottom
     };
 
+    // A coarse grain version of SurfaceClass
```

Comment:
Here is a new `enum` that is useful in daylighting calculations. I think it may be useful elsewhere too so I put it in `DataSurfaces.hh` rather than `DataDaylighting.hh`.

Reply:
I know we are not historically known for good variable names, so I don't have much room to talk.  But FWC feels a little bit too short.  I'm not suggesting we have to change it, or need to do it here, but it might be worth a quick thought about a better name.

Reply:
It was late.

Reply:
SurfLocation, InteriorSurf, SurfOrientation, SurfType, TypeOfSurf

Reply:
You will never see me use `Of` in a variable or type name. EnergyPlus is the only place I have ever seen this.

The thing about `FWC`, is that it's not used for state variables, so there will not be instances of `fwc` out of context. It will always be either `X[(int)FWC::Floor]` or a loop indexed by `fwc`, and mostly the former so a short descriptor is preferred. I agree that `FWC` is not great, but it is short and it should be obvious from context what it is. If people still have heartburn about this, I will change it in the next pass.